pkg/shell/superuser.jsx

```diff
@@ -180,6 +191,33 @@
             <Button variant="secondary" className='btn-cancel' onClick={cancel}>
                 {_("Close")}
             </Button>);
+    } else if (methods) {
+        const apply = () => {
+            start(method);
+        };
+
+        title = _("Switch to administrative access");
+        body = (
+            <Form isHorizontal>
+                <FormGroup fieldId="switch-to-admin-access-bridge-select"
+                           label={_("Method")}>
+                    <FormSelect value={method} onChange={setMethod} isDisabled={busy}>
+                        { methods.map(m => <FormSelectOption value={m} key={m}
+                                                             label={_(proxy.Methods[m].v.label.v)} />) }
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220713-085711-6af71fce-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)